bundle/phases/initialize.go

```diff
@@ -34,6 +35,7 @@ func Initialize() bundle.Mutator {
 			mutator.ExpandPipelineGlobPaths(),
 			mutator.TranslatePaths(),
 			python.WrapperWarning(),
+			permissions.ApplyBundlePermissions(),
```

Comment:
Running this from initialize makes sense, but the workspace path permissions should only be applied after we know for sure the remote workspace path exists. We know this for sure when we run a deploy. The initialize phase should not have any side effects.

Reply:
Agree! Moved workspace root permissions to separate mutator in deploy phase